conda.recipe/run_test.py

```diff
@@ -0,0 +1 @@
+import datashader; datashader.test()
```

Comment:
Ok, that makes sense. Just to clarify, these are only run automatically on build, not on install, correct?